core/http/api/v2/cluster/routers.go

```diff
@@ -124,6 +124,10 @@ func RegisterRoutes(engine *gin.Engine, api *API) {
 			Method:      http.MethodGet,
 			Pattern:     fmt.Sprintf("/clusters/:%v/containers", common.ParamClusterID),
 			HandlerFunc: api.GetContainers,
+		}, {
+			Method:      http.MethodPost,
+			Pattern:     fmt.Sprintf("/clusters/:%v/favorite", common.ParamClusterID),
```

Comment:
Would it be better to split it into two APIs, POST and DELETE?

Reply:
done